gems/smithy/lib/smithy/views/client/request_response_example.rb

```diff
@@ -111,6 +111,17 @@ def structure(structure_shape, indent, visited)
           lines.join("\n")
         end
 
+        def union(union_shape, indent, visited)
+          lines = []
+          lines << '{'
+          union_shape['members']&.first&.tap do |member_name, member_shape|
```

Comment:
Instead of first, I would instead list all of the members, with a top level comment of "One of: " like we do with enums?

Reply:
Yeah that makes sense, will do.